students/Lincoln_Zhang/Session4/trigrams.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+#
+#Trigrams
+#http://codekata.com/kata/kata14-tom-swift-under-the-milkwood/
+#
+#
+#
+import random
+
+from itertools import islice
+
+words = "I wish I may I wish I might".split()
```

Comment:
For the words, you will want to run this against some of the text linked in the assignment. Try to do it with the sherlock_small.txt

You will want to do some sort of small formatting to make the strings look "nicer"